Update active-directory-v2-scopes.md

richdizz opened the pull request:
adminconsent supports using the "common" authority. Even the example you have in here uses "common". Documentation should indicate this as tenant IS required in some areas (ex: client credential flows) but not here.

azurecla commented:
Hi __@sahaj08__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sahajs). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
@barbkess

richdizz committed:
Update active-directory-v2-scopes.md

adminconsent supports using the "common" authority. Even the example you have in here uses "common". Documentation should indicate this as tenant IS required in some areas (ex: client credential flows) but not here.
